build.gradle

```
@@ -1,7 +1,15 @@
-group 'com.auth0'
+plugins {
+    id 'java'
+    id 'jacoco'
+    id 'com.auth0.gradle.oss-library.java'
+    id "com.jfrog.bintray" version "1.8.4"
```

Please move the version to the `pluginManagement` block.

```suggestion
    id 'com.jfrog.bintray'
```

---

Updated as suggested.